### ISSUE TITLE
Cannot run distributable builds on OSX

### ISSUE THREAD

**Issue by jsantell** [mozilla/tofino#623]
Likely because `node` isn't in the path when just running from finder, as opposed to CLI.

```
My guess is that node is only in your PATH when on the command line but I'm not sure.
```

**Comment by Mossop** [comment]
If it is just a missing node then #610 should fix it, I hope to have a fix for that today.

**Comment by jsantell** [comment]
Works when passing in an absolute path to my node executable, so hopefully #610 handles that

**Comment by jsantell** [comment]
Fixed in #610
